spec/requests/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+    let(:valid_attributes) {
```

Comment:
I'm a little confused why we use FactoryBot in the model test but redefine the test data here for the requests.